src/targets/services/cliskTimeout.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * This service stops the given job with a 'context deadline exceeded' error. It may be called when
+ * the execution of the clisk konnector is too long or when the flagship app has crashed.
+ * The flagship app will also timeout the job if it restarts but this way, the user will not see
+ * the job as running even outside the flagship app
+ *
+ * This service is supposed to be called directly or via a @in or @at trigger
+ *
+ * Example to call this service :
+ *
+ *
+ *  await client.collection('io.cozy.jobs').create(
+ *     'service',
+ *     {
+ *       name: 'cliskTimeout',
+ *       slug: 'home',
+ *       fields: {
+ *        cliskJobId: '<the job id>'
+ *       }
+ *     }
+ *   )
+ */
+import CozyClient from 'cozy-client'
+import logger from 'cozy-logger'
+import polyfillFetch from './polyfillFetch'
+
+const log = logger.namespace('cliskTimeout')
+
+polyfillFetch()
+
+const TIMEOUT_ERROR = 'context deadline exceeded' // same error as cozy-stack when interrupting a job for timeout
+const JOBS_DOCTYPE = 'io.cozy.jobs'
+
+const main = async () => {
+  const client = CozyClient.fromEnv()
+
+  const { cliskJobId } = getServiceFields()
```

Comment:
are you saying that we created this variable somewhere in the job? Can't we just use the JOB_ID ?

Reply:
Oh I think I finally understand: 
- Flagship App creates a job for the clisk 
- Flagship App get this clisk job and create a new service (cliskTimeout) with the clisk job to watch 
- If the cliskTimeout service is called, then it will update the clisk job with the right status.